contrib/tidy_datadir.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+# Copyright (c) 2013 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
Should be an https URL, and similarly for other cases.

Reply:
I'm going to address these in another PR. We're going to keep getting these from upstream PRs, so may as well change them periodically in bulk.